scripts/mkcloud

```diff
@@ -326,7 +329,8 @@ function sshrun
         export test_internet_url=$test_internet_url;
         export extraipmipw=$extraipmipw;
         export ipmi_ip_addrs=$ipmi_ip_addrs;
-
+        export bmc_user=$bmc_user;
+        export bmc_password=$bmc_password;
```

Comment:
should be moved to mkcloud-common, so that is also defined for those cases that run without mkcloud.